Yosemite/Yosemite/Stores/ProductStore.swift

```diff
@@ -541,17 +536,15 @@ private extension ProductStore {
                                        url: String,
                                        name: String,
                                        description: String,
-                                       languageCode: String,
                                        completion: @escaping (Result<String, Error>) -> Void) {
         let prompt = [
-            "Your task is to help a merchant create a message to share with their customers the below product.",
-            "Product name: \(name)",
-            "Product description: \(description)",
+            "Your task is to help a merchant create a message to share with their customers a product named \(name). More information about the product:",
+            "Product description: \"\(description)\"",
```

Comment:
Similar to above, putting the variables in some form of marks would help the model distinguish them from the rest of the prompt.

Reply:
Also done in 2e1ba12bcf59b0e409f0a606fdb93ebdbdbf378b.